experimental/SymmetricIntersections/Elevators.jl

```diff
@@ -0,0 +1,445 @@
+import Base: length, IteratorSize, eltype
+
+export associated_bounds,
+       associated_function,
+       degree_of_elevations,
+       elevator,
+       number_of_elevations,
+       underlying_iterator,
+       underlying_list
+"""
+Given a sorted list of integers `L` and an integer `d`, I call a `d`-elevation
+of `L` any set of indices for which the sum of the corresponding elements in `L`
+is `d`. The set of all `d`-elevations of `L` is called the `d`-elevator of `L`.
+For any `d`-elevations `e` of `L`, `d` is called the degree of `e`.
+
+If `(L, f)` consists of a list `L` of objects of the same type and a ``\mathbb Z``-
+valued function `f` such that `f(L)` is a sorted list of integers, the `d`-elevator
+of `(L, f)` is defined to be the `d`-elevator of `f(L)`.
+"""
```

Comment:
This is a free-floating docstring, attached to nothing. As far as I know, there is no way to access this in any way, is there?

Perhaps it would be better to attache this to something?